isso/tests/test_comments.py

```diff
@@ -604,6 +604,115 @@ def testAddComment(self):
         self.app.db.comments.activate(1)
         self.assertEqual(self.client.get('/?uri=test').status_code, 200)
 
+    def testModerateComment(self):
+
+        id_ = 1
+        signed = self.app.sign(id_)
+
+        # Create new comment, should have mode=2 (pending moderation)
+        rv = self.client.post(
+            '/new?uri=moderated', data=json.dumps({"text": "..."}))
```

Comment:
Maybe use `uri=/moderated` (with slash) so that later on, the assert with `invalid.local/moderated` instead of `invalid.localmoderated` is not as confusing